docs/workflows/best-practices-osh.md

```diff
@@ -0,0 +1,398 @@
+  - ​​Open source dependencies can be compared in terms of activity on tools like [https://www.openhub.net/](https://www.openhub.net/)​
+  - Looking at the bug reports of inactive libraries, and the fixed bugs or improvements of outdated libraries can give good information of the relevance of updating.
+
+When many libraries require (multiple or major) version updates, the level of test coverage of a system can be used as an additional factor for prioritization: systems with high test coverage have a lower risk  of running into defects at run-time due to incompatible updates.  
+  
+
+
+## Ensuring your open source stays healthy
+In this section, we are describing guidelines, hints and tips on how to maintain healthy use of open source libraries in your application. Where applicable, we explain how Sigrid can be used to achieve this.
+
+
+### 4. Scan the software for health issues [ROUGH DRAFT]
+- Frameworks and libraries should be scanned frequently (preferably daily, but no less than every 3 months) to make sure there are no severe vulnerabilities in them.
+- Use Sigrid to continuously review libraries:
+  - For existence of known vulnerabilities.
+  - For risks in licenses
+  - For up-to-date-ness (freshness)
+- When to use the scan results: 
+  - Do triage of scan results during refinement. You need to decide if libraries need to be updated during the upcoming sprint, either because vulnerabilities have been found or because the team agrees it is falling too far behind the latest version. 
+  - A licensing risk will usually only appear when a library has been newly introduced.
+  Likely actions:
+    - [Updating a library](#updating-a-library) 
+
+
+### 5. Handling detected vulnerabilities
+Security risks of a certain framework or library should be determined based on at least the following aspects: 
+​- The severity level of known vulnerabilities for the artifact​
+- The mission of the components where the framework or the library will be applied​: Non mission critical / ​Mission ​critical
+- The outside visibility: ​External Facing or Distributed​ 
+  > [? is this the right classification?]
+  - public facing systems need to be tackled first, and should not have any vulnerabilities of risk category _high_ and higher.
+
+When a vulnerability is found, it will be remediated within a specified time period.
+  The table below is a suggestion how to specify this, depending on the criticality of the system:
+  | CVSSv3 Range | Label | Remediation Deadline Critical | Remediation Deadline Regular |
+  | --- | --- | --- | --- |
+  | 9.0 – 10.0 | Critical | Within 1 working day | Within 1 working day | 
+  | 7.0 – 8.9 | High | Within 14 days | Within 30 days |
+  | 4.0 – 6.9 | Medium | Within 60 days | Within 90 days |
+  | 0.1 – 3.9 | Low | Within 1 year | Within 1 year |
+  > TODO: check that the above table makes sense, especially the added last column
```

Comment:
I think you only need this table for public-facing systems, with just one column (like we use it ourselves)

Reply:
so, it does not matter otherwise?? I think security best-practice is that also non-public facing systems need to be secure (multiple lines of defense).